crates/shared/src/zeroex_api.rs

```diff
@@ -174,7 +174,7 @@ impl Default for OrdersQuery {
 #[derivative(Default)]
 #[serde(rename_all = "camelCase")]
 pub struct OrderMetadata {
-    #[derivative(Default(value = "chrono::MIN_DATETIME"))]
+    #[derivative(Default(value = "DateTime::<Utc>::MIN_UTC"))]
```

Comment:
`chrono` got bumped as a result of the crate refactor, and `chrono::MIN_DATETIME` got marked as deprecated